nautobot/extras/tests/test_webhooks.py

```diff
@@ -64,7 +72,105 @@ def mock_send(_, request, **kwargs):
             self.assertEqual(body["model"], "site")
             self.assertEqual(body["username"], "testuser")
             self.assertEqual(body["request_id"], str(request_id))
+            self.assertEqual(body["data"]["name"], "Site Update")
+            self.assertEqual(body["snapshot"]["prechange"]["name"], "Site 1")
+            self.assertEqual(body["snapshot"]["postchange"]["name"], "Site Update")
+            self.assertEqual(body["snapshot"]["differences"]["removed"]["name"], "Site 1")
+            self.assertEqual(body["snapshot"]["differences"]["added"]["name"], "Site Update")
```

Comment:
Can you extend these tests to look a bit more deeply at the snapshot data, at least setting/updating and then checking one or two of the nested serialized fields such as `status` or `region`?